### PR TITLE
Use latest modal version in example

### PR DESCRIPTION
**Pull request by nityas**

### CODE REVIEW

**Comment by tbosch82** [comment]
@nityas Adding `import '@usecapsule/react-sdk/styles.css';` to the root `index.js` should fix your css issues.

**Review by tbosch82 — APPROVED** [review]
lgtm!

File: src/pages/home/components/MainContent/MainContent.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { useState, useEffect } from "react";
 import { OAuthMethod } from "@usecapsule/react-sdk";
-import { CapsuleModal, Theme } from "@usecapsule/react-sdk";
+import { CapsuleModal  } from "@usecapsule/react-sdk";
```
Comment (nityas) [inline src/pages/home/components/MainContent/MainContent.tsx]:
```suggestion
import { CapsuleModal } from "@usecapsule/react-sdk";
```

**Review by vr2amesh — APPROVED** [review]
Looks great!

**Pull request merged**